check_prime.py

```diff
@@ -0,0 +1,41 @@
+# Author:   Tan Duc Mai
+# Email:    tanducwork@gmail.com
+
+"""
+Three different functions to check whether a given number is a prime.
+Return True if it is a prime, False otherwise.
+Those three functions, from a to c, decreases in efficiency (takes longer time).
+"""
+
+
+def is_prime_a(n):
+    from math import sqrt

```

Comment:
should on the top

Reply:
That's a good idea. I'll fix it.
Thanks.